wurst/assets/LocalObjectIDs.wurst

```diff
@@ -36,189 +36,175 @@ public let ABILITY_ATTACK_SPEED_25                     = compiletime(ABIL_ID_GEN
-public let ABILITY_OMNICURE                            = 'OCUR'..registerObjectID("ABIL_OMNICURE")
+public let ABILITY_OMNICURE                            = compiletime(ABIL_ID_GEN.next())..registerObjectID("ABIL_OMNICURE")
 public let ABILITY_OMNICURE_ITEM                       = compiletime(ABIL_ID_GEN.next())..registerObjectID("ABIL_OMNICURE_ITEM")
-public let ABILITY_OMNIGATHERER                        = 'S01M'..registerObjectID("ABIL_OMNIGATHERER")
-public let ABILITY_OMNIGATHERER_INHERITED_SPELLBOOK    = 'OGIS'..registerObjectID("ABIL_OMNIGATHERER_INHERITED_SPELLBOOK")
-public let ABILITY_OMNIGATHERER_MIX_HERBS              = 'MIXO'..registerObjectID("ABIL_OMNIGATHERER_MIX_HERBS")
+public let ABILITY_OMNIGATHERER                        = compiletime(ABIL_ID_GEN.next())..registerObjectID("ABIL_OMNIGATHERER")
+public let ABILITY_OMNIGATHERER_INHERITED_SPELLBOOK    = compiletime(ABIL_ID_GEN.next())..registerObjectID("ABIL_OMNIGATHERER_INHERITED_SPELLBOOK")
+public let ABILITY_OMNIGATHERER_MIX_HERBS              = compiletime(ABIL_ID_GEN.next())..registerObjectID("ABIL_OMNIGATHERER_MIX_HERBS")
 public let ABILITY_TOGGLE_PET_CONTROL                  = compiletime(ABIL_ID_GEN.next())..registerObjectID("ABIL_TOGGLE_PET_CONTROL")
 public let ABILITY_TOGGLE_PET_CONTROL_SHAPESHIFTER     = compiletime(ABIL_ID_GEN.next())..registerObjectID("ABIL_TOGGLE_PET_CONTROL_SHAPESHIFTER")
-public let ABILITY_OVERCHARGE                          = 'ESOC'..registerObjectID("ABIL_OVERCHARGE")
-public let ABILITY_PACK_BUILDING                       = 'A003'..registerObjectID("ABIL_PACK_BUILDING")
+public let ABILITY_OVERCHARGE                          = compiletime(ABIL_ID_GEN.next())..registerObjectID("ABIL_OVERCHARGE")
+public let ABILITY_PACK_BUILDING                       = compiletime(ABIL_ID_GEN.next())..registerObjectID("ABIL_PACK_BUILDING")
 public let ABILITY_PAIN_AURA                           = compiletime(ABIL_ID_GEN.next())..registerObjectID("ABIL_PAIN_AURA")
-public let ABILITY_PANIC                               = 'A010'..registerObjectID("ABIL_PANIC")
-public let ABILITY_PANTHER_PROWL                       = 'A0GF'..registerObjectID("ABIL_PANTHER_PROWL")
+public let ABILITY_PANIC                               = compiletime(ABIL_ID_GEN.next())..registerObjectID("ABIL_PANIC")
+public let ABILITY_PANTHER_PROWL                       = compiletime(ABIL_ID_GEN.next())..registerObjectID("ABIL_PANTHER_PROWL")
 public let ABILITY_PET_ARMOR                           = compiletime(ABIL_ID_GEN.next())..registerObjectID("ABIL_PET_ARMOR")
-public let ABILITY_PET_ATTACK                          = 'A01B'..registerObjectID("ABIL_PET_ATTACK")
-public let ABILITY_PET_BRING_ITEMS                     = 'A06R'..registerObjectID("ABIL_PET_BRING_ITEMS")
-public let ABILITY_PET_DAMAGE                          = compiletime(ABIL_ID_GEN.next())..registerObjectID("ABIL_PET_DAMAGE")
```

Comment:
Isn't this still used in some definitions, such as ElkDefinition.wurst?

Reply:
You are right, I probably removed it by inadvertence during merging.